assets/css/new-dotorg-design.css

```diff
@@ -0,0 +1,50 @@
+body {
+	--wp--preset--color--charcoal-2: #23282d;
+	--wp-global-header--background-color: var(--wp--preset--color--charcoal-2);
+	--wp--preset--color--charcoal-4: #656a71;
+	--wp--preset--color--white-opacity-15: #ffffff26;
+}
```

Comment:
I'm assuming you copied this CSS from somewhere else, could we add an inline comment that points to the original source?

I think once we have a first version of the design, we should look into how can not duplicate CSS of the theme, so it will be useful to know where the CSS came from.

Reply:
I added the URL in the header of the file.